scrapers/wv/events.py

```diff
@@ -132,10 +134,28 @@ def scrape_meeting_page(self, url):
                 agenda = event.add_agenda_item(
                     row.text_content().strip().replace("\u25a1", "")
                 )
-                for bill in re.findall(self.bill_regex, row.text_content()):
+
+                # Matches (SJR, HCR, HB, HR, SCR, SB, HJR, SR) + id
+                # Allows for house, senate, joint, or bill to be fully spelled out
+                # Allows for "." after H, S, J, C, and B
+                # Allows for up to two spaces before the id
+                bills = re.findall(
+                    r"((S\.?|Senate|H\.?|House)\s?((J|C|Joint)\.?\s?)?(B\.?|Bill|R\.?)\s?\s?(\d+))",
+                    row.text_content(),
+                    flags=re.IGNORECASE,
+                )
+
+                for bill in bills:
                     bill_id = re.sub(r"\.\s*", "", bill[0], flags=re.IGNORECASE)
                     bill_id = re.sub(r"house bill", "HB", bill_id, flags=re.IGNORECASE)
                     bill_id = re.sub(r"senate bill", "SB", bill_id, flags=re.IGNORECASE)
+
+                    # Final step to set correct number of spaces in the id
+                    components = re.search(
+                        r"([A-Z]+)\s*(\d+)", bill_id, flags=re.IGNORECASE
+                    )
+                    bill_id = f"{components.group(1)} {components.group(2)}"
```

Comment:
Need to adjust the regex to exclude any leading zeros from the numerical component of the `bill_id`.
- i.e. "SB 009" should be "SB 9", "HR 0243" should be "HR 243"

Reply:
Fixed with https://github.com/openstates/openstates-scrapers/pull/4521/commits/4ffa4ce024bf4158035c68261f91b97a089860cb